lib/fs.js

```diff
@@ -2042,10 +2045,7 @@ function ReadStream(path, options) {
     this.pos = this.start;
   }
 
-  // Case 0: If end is undefined, set end to Infinity.
-  if (this.end === undefined) {
-    this.end = Infinity;
-  } else {
+  if (this.end !== undefined) {
```

Comment:
It seems that `this.end` will never be `undefined` here.

Reply:
It won't. I realize what's wrong here, will change this clause. Thanks for pointing out.